install_requirements.py

```diff
@@ -101,34 +101,16 @@ def install_requirements(use_pytorch_nightly):
             if use_pytorch_nightly
             else "torchvision"
         ),  # For testing.
-        "typing-extensions",
     ]
 
-    # pip packages needed to run examples.
-    # TODO: Make each example publish its own requirements.txt
     EXAMPLES_REQUIREMENTS = [
-        "timm==1.0.7",
         f"torchaudio==2.6.0.{NIGHTLY_VERSION}" if use_pytorch_nightly else "torchaudio",
-        "torchsr==1.0.4",
-        "transformers==4.47.1",
-    ]
-
-    # pip packages needed for development.
-    DEVEL_REQUIREMENTS = [
-        "cmake",  # For building binary targets.
-        "pip>=23",  # For building the pip package.
-        "pyyaml",  # Imported by the kernel codegen tools.
-        "setuptools>=63",  # For building the pip package.
-        "tomli",  # Imported by extract_sources.py when using python < 3.11.
-        "wheel",  # For building the pip package archive.
-        "zstd",  # Imported by resolve_buck.py.
```

Comment:
do these need to be added to requirements-dev.txt?

Reply:
no. they are in the pyproject.toml already.